featuretools/tests/entityset_tests/test_serialization.py

```diff
@@ -139,13 +163,28 @@ def test_to_parquet(es, tmpdir):
     assert type(new_es['log'].df['latlong'][0]) == tuple
 
 
+def test_dask_to_parquet(dask_es, tmpdir):
+    dask_es.to_parquet(str(tmpdir))
+    new_es = deserialize.read_entityset(str(tmpdir))
+    assert dask_es.__eq__(new_es, deep=True)
+    assert type(dask_es['log'].df.set_index('id')['latlong'].compute()[0]) == tuple
+    assert type(new_es['log'].df.set_index('id')['latlong'].compute()[0]) == tuple
+
+
 def test_to_parquet_manual_interesting_values(es, tmpdir):
     es['log']['product_id'].interesting_values = ["coke_zero"]
     es.to_pickle(str(tmpdir))
     new_es = deserialize.read_entityset(str(tmpdir))
     assert es.__eq__(new_es, deep=True)
 
 
+def test_dask_to_parquet_manual_interesting_values(dask_es, tmpdir):
+    dask_es['log']['product_id'].interesting_values = ["coke_zero"]
+    dask_es.to_pickle(str(tmpdir))
```

Comment:
This test is using `to_pickle`. Should this be `to_parquet` based on the test name? Also appears to be an issue with `test_to_parquet_manual_interesting_values` above. For the deserialize operation below, the resulting entityset contains pandas dataframes, but the equality assertion still works.

Reply:
Fixed both tests. It's interesting that a dask/pandas entityset would be considered deeply equal, that's probably an issue that should be addressed separately.